src/Console/DuskCommand.php

```diff
@@ -282,7 +283,11 @@ protected function writeConfiguration()
     {
         if (! file_exists($file = base_path('phpunit.dusk.xml')) &&
             ! file_exists(base_path('phpunit.dusk.xml.dist'))) {
-            copy(realpath(__DIR__.'/../../stubs/phpunit.xml'), $file);
+            if (version_compare(Application::VERSION, '10.0.0') >= 0) {
```

Comment:
```suggestion
            if (version_compare(Application::VERSION, '10.0', '>=')) {
```

Reply:
But this doesn't guarantee PHPUnit 10 is installed, maybe use `version_compare()` against phpunit instead?